README.md

```diff
@@ -6,7 +6,7 @@ eBPF-based autoinstrumentation of HTTP/HTTPS/GRPC Go services, as well as HTTP/H
 written in other languages (intercepting Kernel-level socket operations as well as
 OpenSSL invocations).
 
-[Documentation](./docs).
+[Documentation](https://grafana.com/docs/TODO).
```

Comment:
Good point. We started the conversation on naming few weeks ago, we'll come up with a name and re-brand everything by end of the month.